### PR TITLE
Add Release Notes page

### LINKED ISSUE #294: Add Release Notes page
See [JxBrowser Release Notes](https://jxbrowser-support.teamdev.com/release-notes/) for now it should look and work.

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR closes #294 

This PR brings a new Release Notes page with all releases since 2015.
Also, the `Versioning Policy` page was added.

### CODE REVIEW

**Comment by JuliaEvseeva** [comment]
@andreySadomovskiy, PTAL.
@oleksandra-kovalenko, please take a look at the content text.

File: _sass/base/_markdown.scss [hunk 1]
```diff
@@ -108,6 +113,12 @@ $article-line-height: 1.74;
     margin: 8px 0 24px;
   }
 
+  .date {
+    font-family: $main-font;
+    font-size: 14px;
+    color: rgba(black, .54);
```
Comment (andreySadomovskiy) [inline _sass/base/_markdown.scss:119]: I found several usages of this `color: rgba(black, .54);` color and `color: rgba(black, .38);`. Maybe better to move them to variables. What do you think?

Comment (JuliaEvseeva) [reply _sass/base/_markdown.scss:119]: Will be done with #327.

**Comment by JuliaEvseeva** [comment]
@andreySadomovskiy, PTAL again. The `Versioning Policy` page was moved to the `versioning` folder and the `privacy` layout was modified to use it for this page.

**Comment by andreySadomovskiy** [comment]
@JuliaEvseeva , I have not any comments.  LGTM

**Comment by oleksandra-kovalenko** [comment]
@JuliaEvseeva LGTM

**Pull request merged**